optuna/storages/_rdb/storage.py

```diff
@@ -900,8 +900,8 @@ def get_all_trials(
     def _get_trials(
         self,
         study_id: int,
-        states: Optional[Tuple[TrialState, ...]],
         excluded_trial_ids: Set[int],
+        states: Optional[Container[optuna.trial._state.TrialState]] = None,
```

Comment:
```suggestion
        states: Optional[Container[optuna.trial._state.TrialState]],
        excluded_trial_ids: Set[int],
```

Please do not change the order of arguments. `_get_trials` is internal functionality so I think we don't have to set a default value.